packages/snaps-utils/src/platform-version.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Get the current supported platform version.
+ *
+ * Note: This function assumes that the same SDK version is used across all
+ * dependencies. If this is not the case, the version of the SDK that is
+ * closest to the `snaps-utils` package will be returned.
+ *
+ * @returns The platform version.
+ */
+export function getPlatformVersion() {
+  const packageJson = require.resolve('@metamask/snaps-sdk/package.json');
+  // eslint-disable-next-line import/no-dynamic-require, @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+  return require(packageJson).version;
```

Comment:
Does this work in Browserify?

Reply:
It may not, but I realised we can simplify this to `require('@metamask/snaps-sdk/package.json').version`, which should definitely work in Browserify.